torch/_numpy/_funcs_impl.py

```diff
@@ -288,8 +290,8 @@ def vander(x: ArrayLike, N=None, increasing=False):
 
 
 def linspace(
-    start: ArrayLike,
-    stop: ArrayLike,
+    start: ArrayLikeOrScalar,
+    stop: ArrayLikeOrScalar,
```

Comment:
```suggestion
    start,
    stop,
```
same reason as below.